cypress/e2e/Editor/PortfolioEditor.cy.js

```diff
@@ -0,0 +1,82 @@
+// import {signIn} from '../DoenetSignin/DoenetSignin.cy';
+
+const { cesc2 } = require("../../../src/_utils/url");
+
+describe("Share Activities Using Portfolio", function () {
```

Comment:
Give it a relevant name such as "Portfolio editor tests".

While you're at it, can you also rename PortfolioVariants.cy.js, ActivtyControls.cy.js, and LoadAndSaveEditor.cy.js? It seems we are both in the habit of copying over test files without renaming them!